generate_R_table.py

```diff
@@ -1,3 +1,4 @@
+#!/Users/antonmalko/anaconda/bin/ python
```

Comment:
I mean, I was mostly coding it for myself - the script also assumes a certain folder structure which I don't specify anywhere. But yeah, in terms of Python interpreter I can try to make it more portable. 

Didn't realize you would receive notifications about changes, I was just playing with this code to adapt it better to what I need now =)